app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -25,11 +25,21 @@ def self.rrd_claim_processed?(submission)
       submission.form_json.include? RapidReadyForDecision::HypertensionUploadManager::DOCUMENT_TITLE
     end
 
+    # Fetch all claims from EVSS and return whether there are any open EP 020's.
+    # This method could be moved into a Concern when ProcessorSelector adds new job classes.
+    def self.pending_eps?(auth_headers)
+      all_claims = EVSS::ClaimsService.new(auth_headers).all_claims.body
+      all_claims['open_claims'].any? { |claim| claim['base_end_product_code'] == '020' }
+    end
+
     def perform(form526_submission_id)
       form526_submission = Form526Submission.find(form526_submission_id)
 
       begin
         with_tracking(self.class.name, form526_submission.saved_claim_id, form526_submission_id) do
+          # Prefer "next" keyword here so that after-yield code continues execution
+          next if pending_eps?(form526_submission.auth_headers)
```

Comment:
Tests fail here because I need to wrap relevant spec in the cassette code. Can this be done in a separate PR @yoomlam?

Additionally, I'm finding I have to reference to this class method as `RapidReadyForDecision::Form526BaseJob` which... makes sense. The job method context is apparently divorced from class methods too.

Reply:
You'll need to get all tests passing in order to merge this PR, so you'll have to address the failing tests in this PR.

Reply:
@yoomlam I know that tests need to pass to merge PRs. I'm asking if moving the fix to the base class into a separate PR would make sense.

Reply:
Since it's a bug that requires a small fix, I'd apply fix ASAP. Up to you.

Reply:
@yoomlam yep, it's already applied. Can I get an approve?